pkg/driver/bigquery/bigquery_test.go

```diff
@@ -0,0 +1,252 @@
+package bigquery
+
+import (
+	"database/sql"
+	"net/url"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/amacneil/dbmate/v2/pkg/dbmate"
+	"github.com/amacneil/dbmate/v2/pkg/dbutil"
+)
+
+func getURL() string {
+	return os.Getenv("BIGQUERY_TEST_URL")
+}
+
+func testBigQueryDriver(t *testing.T) *Driver {
+	u := dbutil.MustParseURL(getURL())
+	drv, err := dbmate.New(u).Driver()
+	require.NoError(t, err)
+
+	return drv.(*Driver)
+}
+
+func prepTestBigQueryDB(t *testing.T) *sql.DB {
+	drv := testBigQueryDriver(t)
+
+	// drop any existing database
+	err := drv.DropDatabase()
+	require.NoError(t, err)
+
+	// create database
+	err = drv.CreateDatabase()
+	require.NoError(t, err)
+
+	// connect database
+	db, err := drv.Open()
+	require.NoError(t, err)
+
+	return db
+}
+
+func TestGetDriver(t *testing.T) {
+	db := dbmate.New(dbutil.MustParseURL("bigquery://test/db_mate"))
+	drvInterface, err := db.Driver()
+	require.NoError(t, err)
+
+	// driver should have URL and default migrations table set
+	drv, ok := drvInterface.(*Driver)
+	require.True(t, ok)
+	require.Equal(t, db.DatabaseURL.String(), drv.databaseURL.String())
+	require.Equal(t, "schema_migrations", drv.migrationsTableName)
+}
+
+func TestConnectionString(t *testing.T) {
+	cases := []struct {
+		input    string
+		expected string
+	}{
+		{"bigquery://projectid/dataset", "bigquery://projectid/dataset?disable_auth=false"},
```

Comment:
Assuming that most users of dbmate will be using it with Google's actual BigQuery service and not the emulator, I don't think `disable_auth=false` should be the implied default.  For this input, I think `bigquery://projectid/dataset` should be the expected connection string.  What do you think?